x/mongo/driver/topology/connection_test.go

```diff
@@ -839,18 +839,19 @@ func TestConnection(t *testing.T) {
 			makeMultipleConnections := func(t *testing.T, numConns int) (*pool, []*Connection) {
```

Comment:
[opt] It could be easier to have `makeMultipleConnections`, `makeOneConnection` and `createTestPool` also return the `pool.disconnect` function so we never forget to defer it. I gather this is a common pattern for `t.Helper()` functions that instantiate resources.

Reply:
That sounds like a good idea, will update.